spec/features/projects/modules_spec.rb

```diff
@@ -30,43 +30,37 @@
 
 RSpec.describe 'Projects module administration' do
   let!(:project) do
-    create(:project,
-           enabled_module_names: [])
+    create(:project, enabled_module_names: [])
   end
 
-  let(:permissions) { %i(edit_project select_project_modules) }
+  let(:permissions) { %i(edit_project select_project_modules view_work_packages) }
   let(:settings_page) { Pages::Projects::Settings.new(project) }
 
   current_user do
     create(:user, member_with_permissions: { project => permissions })
   end
 
   it 'allows adding and removing modules' do
+    project_work_packages_menu_link_selector = '//ul[contains(@class, "menu_root")]//span[text()="Work packages"]'
```

Comment:
Should this be a `let`?

Reply:
It can be. I preferred local variable, because the value has been used only in one example. I would leave it like that for now if you do not mind.

Reply:
Sure, it is a nitpick anyway